certified-connectors/HrFlow/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "Get an API Key from HrFlow.ai",
+          "tooltip": "Provide your API Key - learn more https://developers.hrflow.ai/getting-started/authentication",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFE",
```

Comment:
@alaeddinechhima White background color is not allowed, can you please follow the instructions mentioned in the documentation for icon and update it as per the new icon's background color.

Docs - [Icon](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon)

Reply:
Thank you for the feedback. 

I pushed an update the icon color background and action's summary.